Source/WebCore/platform/graphics/x11/XUniquePtr.h

```diff
@@ -27,13 +27,18 @@
 #define XUniquePtr_h
 
 #if PLATFORM(X11)
+
 #include "PlatformDisplayX11.h"
 #include <X11/Xutil.h>
 
 #if USE(GLX)
+#if USE(LIBEPOXY)
+#include <epoxy/glx.h>
+#else
```

Comment:
Do we really need this here? It's problematic to include platform gl headers from .h files.

Reply:
It is needed, but it's avoidable if the template definition is moved into a .cpp file that has to be created.